modules/ctl/src/main/scala/hi/deploy.scala

```diff
@@ -37,7 +42,7 @@ object deploy {
         c <- info(s"Using $rev.") *> commitForRevision(rev)
         _ <- info(s"Commit is ${c.hash}")
         u <- uncommittedChanges.map(_ && rev === "HEAD")
-        _ <- u.whenM(warn("There are uncommitted changes. This is a UNCOMMITTED deployment."))
+        _ <- if (u) warn("There are uncommitted changes. This is a UNCOMMITTED deployment.") else ().pure[CtlIO]
```

Comment:
Just re-read the top level comment and realized it's coming the next version, so never mind.